src/rules/padding.ts

```diff
@@ -0,0 +1,438 @@
+/**
+ * @fileoverview Rule to require or disallow newlines between jest functions,
+ *   based on eslint/padding-line-between-statements by Toru Nagashima
+ * @author Ben Kimpel
```

Comment:
We can remove me as the author here... I mostly just removed things from this file.